backend/app/configs/data_saver/sqlite_saver.yaml

```
@@ -4,4 +4,4 @@ defaults:
 
 name: sqlite_saver
 source: $kafka
-sqlite_db: ${oc.env:ROOT_PATH}/data_layer/database/db/sqlite
\ No newline at end of file
+sqlite_db: ${oc.env:ROOT_PATH}/app/data_layer/database/db/sqlite/sqlite_db
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)